tst/testinstall/methwhy.tst

```diff
@@ -35,52 +37,90 @@ function ( x )
 end
 gap> Display(ApplicableMethod(foobar, [ ['a'] ], 1));
 #I  Searching Method for foobar with 1 arguments:
-#I  Total: 4 entries
-#I  Method 1: ``foobar: for a string'' at stream:1 , value: 2*SUM_FLAGS+1
+#I  Total: 4 entries, of which 3 are applicable:
+#I  Method 1, valid operation number 1, value: 2*SUM_FLAGS+1
+#I  ``foobar: for a string''
+#I   at stream:1
 function ( x )
     return StartsWith( x, "foo" );
 end
-gap> Display(ApplicableMethod(foobar, [ ['a'] ], 1, 2));
+gap> ApplicableMethod(foobar, 'a', 1);   
```

Comment:
Why was the original test replaced by another? It is of course fine to add new tests, but why remove the old?

Reply:
Examples with "..., 1, 3)",  "..., 1, 4)",  "..., 1, "all"). immediately follow, so the "..., 1, 2)" case seemed unnecessary.

Reply:
Well, the idea was to test *all* possible ways to call this function. Why would one randomly skip tests for some ways to call it?